:sparkles: Add support to editable vector joins (1:1 relations)

wlorenzetti opened the pull request:
Closes: #598

## Related

- https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52
- https://github.com/g3w-suite/g3w-client/pull/483

Commit:
Add 'hasCustomPrefix' property to vectorjoin.

Commit:
Add 'editable' property, so serialization of vector join.

Commit:
Add property `vectorjoin_id` to REST API /vector/api/config/.

Commit:
Fix test.

Commit:
Add 'prefix' and convert 'editable' to boolean type.

Commit:
Typo.

Commit:
Fix 'editable' property for fields in QGIS join.

Commit:
Set 'editable' to False if join field is a Pk.

Commit:
Fix test editing fi QGIS Join

Commit:
For documentation.

volterra79 committed:
:sparkles: Client:
   core: https://github.com/g3w-suite/g3w-client/pull/483
   editing: https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52

volterra79 committed:
Merge branch 'update_one_to_one_join_info' of https://github.com/g3w-suite/g3w-admin into update_one_to_one_join_info

volterra79 committed:
:sparkles: Client:
    client:  - https://github.com/g3w-suite/g3w-client/pull/483
             - https://github.com/g3w-suite/g3w-client/pull/159
    editing: - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52
             - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/53

volterra79 committed:
:sparkles: Client:
    client:  - https://github.com/g3w-suite/g3w-client/pull/483
             - https://github.com/g3w-suite/g3w-client/pull/159
    editing: - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52
             - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/53

Commit:
Merge branch 'dev' into update_one_to_one_join_info

# Conflicts:
#	g3w-admin/client/static/client/js/app.min.js
#	g3w-admin/editing/static/editing/js/plugin.js

volterra79 committed:
:sparkles: Client:
    editing: - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52/commits/96c9b400a490a033c85430c92538b91e91e77868
    editing: - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52/commits/16f4c9f855aef6af4818c2a813e43922ad2f5b60
    editing: - https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52/commits/b7a6bb7873ebed5e81754b4b31bf22e8d5466698

volterra79 committed:
:sparkles: Client:
    g3w-client: - https://github.com/g3w-suite/g3w-client/pull/483/commits/5b2e3fae47a9df884b3cf3b7a75fd374de20ff8c

volterra79 committed:
:sparkles: Client:
    g3w-client-plugin-editing: -https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52/commits/cd65314d1b3c064d8dc3c143bfb3c618f550f272

wlorenzetti commented:
@volterra79 please review this and said me if I can merge it on dev.

volterra79 commented:
> @volterra79 please review this and said me if I can merge it on dev.

@wlorenzetti need to work to client branches and i let you know

volterra79 committed:
:sparkles: Client:
    g3w-client:https://github.com/g3w-suite/g3w-client/pull/483/commits/f55754269ba7485dad9591c8ef55cd253cb57259
    g3w-client-plugin-editing: https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52/commits/8aef6a4918eb1dbbdb2e5e44946a1c64db3eb10b

Commit:
Merge branch 'dev' into update_one_to_one_join_info
